packages/ionio/test/e2e/transferWithKey.test.ts

```diff
@@ -21,24 +22,41 @@ describe('TransferWithKey', () => {
       network,
       ecc
     );
-    const response = await faucetComplex(contract.address, 0.0001);
+    // re-using script's pubkey for the blinding one
+    const confidentialAddress = address.toConfidential(
+      contract.address,
+      alicePk.publicKey
+    );
+    const response = await faucetComplex(
+      confidentialAddress,
+      0.0001,
+      network.assetHash,
+      alicePk.privateKey
+    );
 
     prevout = response.prevout;
     utxo = response.utxo;
+    unblindData = response.unblindData || undefined;
   });
 
   describe('transfer', () => {
     it('should transfer with signature', async () => {
       const to = payments.p2wpkh({ pubkey: alicePk.publicKey }).address!;
+      const toConfidential = address.toConfidential(to, alicePk.publicKey);
```

Comment:
This could also be written as:

```suggestion
      const toConfidential = payments.p2wpkh({ pubkey: alicePk.publicKey, blindkey: alicePk.publicKey }).confidentialAddress!;
```